example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -237,9 +237,16 @@ int main(int argc, char **argv)
     const ActionResult land_result = action->land();
     action_error_exit(land_result, "Landing failed");
 
+    // Check if vehicle is still in air
+    while (telemetry->in_air() == true) {
```

Comment:
What about removing `== true`?

```suggestion
    while (telemetry->in_air()) {
```

Reply:
Copy paste from while (telemetry->health_all_ok() != true) frome above. Just remove it.